code/Location.php

```diff
@@ -40,12 +39,12 @@ class Location extends DataObject implements PermissionProvider
         'State',
         'Postcode',
         'Country',
+        'Website',
+        'Phone',
+        'EmailAddress',
```

Comment:
Again, just things that really have no bearing on how it works... Do we want to shorten this to `Email`?